src/components/Message/MessageDeleted.tsx

```diff
@@ -0,0 +1,62 @@
+import React, { useContext } from 'react';
+import type { MessageResponse } from 'stream-chat';
+import type { TranslationContextValue } from 'types';
```

Comment:
we've stopped using the import `from 'types';` and import them from where the type is actually declared now. we're moving towards not having the `index.d.ts`